### PR TITLE
Scale down screenshots when using a retina device on chrome

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

File: lib/applitools/selenium/browser.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+module Applitools::Selenium
+  class Browser
+    JS_GET_USER_AGENT = 'return navigator.userAgent;'.freeze
+    JS_GET_DEVICE_PIXEL_RATIO = 'return window.devicePixelRatio;'.freeze
+    JS_GET_PAGE_METRICS = <<-EOF
```
Comment (bootstraponline) [inline lib/applitools/selenium/browser.rb]: ` JS_GET_PAGE_METRICS = (<<-EOF).freeze`

Comment (lbeder) [inline lib/applitools/selenium/browser.rb]: That's doesn't really freeze it - try it out...

Comment (bootstraponline) [inline lib/applitools/selenium/browser.rb]: ....? I just checked in Pry. I think I'm correct.

```ruby
tmp = (<<EOF).freeze
hello
EOF
tmp[0..1] = '2'
RuntimeError: can't modify frozen String
```

**Comment by bootstraponline** [comment]
I know this isn't appium however a [clean commit history](https://github.com/appium/appium/blob/master/docs/en/contributing-to-appium/style-guide.md#rebasing) would be nice. For example, rebasing out the merge commits.

File: lib/applitools/selenium/browser.rb (hunk 1 above)
Comment (lbeder) [inline lib/applitools/selenium/browser.rb]: Ah, right. I've missed the parentheses.

**Comment by lbeder** [comment]
Thanks for the suggestion! We'll prefer rebasing in the future.

**Comment by bootstraponline** [comment]
Cool. Thanks for being open to feedback.

**Pull request merged**